services/QuillLMS/app/controllers/api/v1/concept_feedback_controller.rb

```diff
@@ -1,15 +1,17 @@
 # frozen_string_literal: true
 
 class Api::V1::ConceptFeedbackController < Api::ApiController
-  before_action :activity_type, except: [:index]
-  before_action :concept_feedback_by_uid, except: [:index, :create, :update]
+  before_action :activity_type, except: [:index, :translations]
+  before_action :concept_feedback_by_uid, except: [:index, :create, :update, :translations]
```

Comment:
This is not your code, but we have moved to generally preferring `only` over `except` so that you don't have to go in and add your new actions to lists like this if the `before_actions` are irrelevant.  Might be worth cleaning up here, but, again, this isn't your code so feel free to skip it.

Reply:
I will make another PR for this, as I am a tiny bit nervous about fat-fingering something and screwing us up. Would rather have that isolated.